packages/engine/paima-rest/src/controllers/AchievementsController.ts

```diff
@@ -1,25 +1,28 @@
 import { Controller, Get, Path, Query, Route } from 'tsoa';
 import { EngineService } from '../EngineService.js';
 import { ENV } from '@paima/utils';
-import type {
-  AchievementService,
-  AchievementPublicList,
-  PlayerAchievements,
-  Validity,
+import {
+  type AchievementPublicList,
+  type PlayerAchievements,
+  type Validity,
+  type Game,
+  type Player,
+  getNftOwner,
 } from '@paima/utils-backend';
+import { getAchievementTypes, getAchievementProgress } from '@paima/db';
 
 // ----------------------------------------------------------------------------
 // Controller and routes per PRC-1
 
-function service(): AchievementService {
-  return EngineService.INSTANCE.achievementService;
-}
-
 @Route('achievements')
 export class AchievementsController extends Controller {
-  private async defaultValidity(): Promise<Validity> {
+  private async game(): Promise<Game> {
+    return { id: 'TODO' };
```

Comment:
TODO

Reply:
Does a DB table with a single row that the game has to fulfill make sense here? Or should there be some code-side meeting point where the game supplies this metadata?